Gemfile

```diff
@@ -22,4 +24,10 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+group :test do
+  gem 'faker', '~> 1.9.4'
```

Comment:
faker should go in development too to be used with factory bot to create random things in console

Reply:
Agree